lib/phoenix_html/form_data.ex

```diff
@@ -52,11 +52,12 @@ defimpl Phoenix.HTML.FormData, for: [Plug.Conn, Atom] do
   def to_form(conn_or_atom, opts) do
     {name, params, opts} = name_params_and_opts(conn_or_atom, opts)
     {errors, opts} = Keyword.pop(opts, :errors, [])
+    {id, opts} = Keyword.pop(opts, :id)
```

Comment:
Hey @josevalim I mess it up. We can't use `Keyword.pop/2` here since we still need to pass opts with id forward 🙈 

The correct would be:

```Elixir
    id = Keyword.get(opts, :id) || name
```

I can open another PR unless you want to fix it right way 😅

Reply:
PR is open: https://github.com/phoenixframework/phoenix_html/pull/307

Reply:
https://github.com/phoenixframework/phoenix_ecto/pull/129